python/src/DualLinearCombinationFunction.i

```diff
@@ -8,3 +8,11 @@
 
 %include openturns/DualLinearCombinationFunction.hxx
 namespace OT { %extend DualLinearCombinationFunction { DualLinearCombinationFunction(const DualLinearCombinationFunction & other) { return new OT::DualLinearCombinationFunction(other); } } }
+
+%pythoncode %{
+def __DualLinearCombinationFunction_repr_html(self):
+    """Get HTML representation."""
+    return self.__repr_html__()
```

Comment:
Once the code is evaluated, please write the HTML code inside the SWIG/Python layer.

Reply:
yes, you had to overload it from c++, so this can be dropped

Reply:
Ok. This is done after 270217be782ff9c0cf28dbb5cb0d519137ede00e.